charts/tekton/templates/ingress.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.dashboard.ingress.enabled -}}
```

Comment:
Does it make sense to have `tekton-dashboard` stuff from `tekton-dashboard.yaml` without adding ingress to use this dashboard? I think that we probably don't need to have a sepatratre `enabled` flag on `dashboard.ingress` level, cause as I see it - it makes no sense to have dashboard without its url address.

Reply:
My rationale behind this decision is that we **may** want to have the dashboard, but not the public ingress. In this case we want developers to access the UI by doing `port-forward`.